packages/mdx-embed/src/components/lbry/lbry.text.tsx

```diff
@@ -0,0 +1,17 @@
+import { act, render, screen } from '@testing-library/react';
```

Comment:
This file should be called `lbry.test.tsx`

Reply:
YIKES! Done!